packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -73,12 +73,17 @@ AppConfig.restore = () => {}
 AppConfig.extraGetPropsArgs = () => {}
 AppConfig.freeze = () => {}
 
-// Recommended settings for PWA-Kit usage.
+const isServerSide = typeof window === 'undefined'
```

Comment:
Currently, the change lives in the test-commerce template only. But how about the other templates (and future ones too)?

If we do want to update them all, should we have the default options extracted and live in the pwa-kit sdk instead?
```js
export const getReactQueryDefaultOptions = () => ...
```

Reply:
I might be misunderstanding but wouldn't you want to expand the options for `commerce-sdk-react` here to include a param like `<CommerceApiProvider reactQueryConfig={<object>} />` and add your unit test coverage there?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/00ac8b6c8f6feb5ec17b4ae70377e0abf1054b68/packages/test-commerce-sdk-react/app/components/_app-config/index.tsx#L31-L41

Reply:
This is a good point. Do I think we are at the point where we have to solve this problem... I don't think so just yet. Is it equally possible that future projects require a different react query configuration.. probably. 

I'd vote to not solve this right now. But if you want an action item for this, maybe we can discuss it in the next eng meeting? What do you think.

Reply:
@bfeister unfortunately, `commerce-sdk-react` no longer is able accept the react-query configuration, after the change made in this PR #734 .

So that the pwa-kit sdk is able to track all queries (used in the project, in commerce-sdk-react, etc) and run them on server side, we require that a single Query Client to be reused. Using the HOC `withReactQuery` and passing in their default options, developers are now in control over the react query configuration for all packages.

Reply:
> Do I think we are at the point where we have to solve this problem... I don't think so just yet.

@bendvc Does that mean that for now we should still update all the other templates? But we'll simply hardcode and duplicate the options?

Reply:
Yes that is what I'm suggesting. That being said there are no other templates to set options for as they aren't using react query at the moment.